src/helpers/figures/charts/chart_common.ts

```diff
@@ -492,7 +499,14 @@ export function getFullTrendingLineDataSet(
   config: TrendConfiguration,
   data: number[]
 ) {
-  const backgroundColor = config.color || lightenColor(dataset.backgroundColor as string, 0.5);
+  const defaultBorderColor = colorToRGBA(dataset.backgroundColor as Color);
+  defaultBorderColor.a = 1;
+
+  const borderColor = config.color || lightenColor(rgbaToHex(defaultBorderColor), 0.5);
+  const backgroundRGBA = colorToRGBA(borderColor);
+  // @ts-expect-error
+  if (dataset?.fill) backgroundRGBA.a = LINE_FILL_TRANSPARENCY; // to support area charts
```

Comment:
aren't you one of those that tell me that one-liners `if` are ugly and hard to read ? :upside_down_face:

Reply:
<3